### ISSUE TITLE
FIXME: "git executable was not found in $Path" despite git being installed

### ISSUE THREAD

**Issue by Zelren** [RimSort/RimSort#382]
**Describe the bug**
Installed git as requested (default directory C:/program files/git) but clicking on either of the download buttons in the databases section brings up the "git executable was not found in $path - would you like to download it?" dialoge.

**To Reproduce**
1. Go to 'Settings- Databases'
2. Click on 'Download'

**Expected behavior**
Rimsort starts downloading the databse from the github repository

**Please complete the following information:**
 - OS: Windows 10
 - RimSort version: 1.0.7
 - Please provide a `RimSort.log`: https://0x0.st/XNlu.log
 - Running offical Rimsort 1.0.7 from the github page

**Comment by hk21702** [comment]
Like the error suggests, check if git is in your path variable. 

Occasionally, you may need to restart your computer after installation for it to work correctly. To check if git is working correctly, just open a command prompt and put in `git -v`.  If this command still does not work after a computer restart, then you need to add git to your path variable. If you don't know how, there are lots of guides online on how to do this.

**Comment by Zelren** [comment]
Thank you, restarting helped. I'd suggest adding this information to the error popup in order to avoid having to deal with this in the future. Have a good day.
